src/androidInstrumentedTest/kotlin/software/momento/kotlin/sdk/CacheClientScalarTest.kt

```diff
@@ -0,0 +1,76 @@
+package software.momento.kotlin.sdk
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import software.momento.kotlin.sdk.exceptions.InvalidArgumentException
+import software.momento.kotlin.sdk.responses.cache.DeleteResponse
+import software.momento.kotlin.sdk.responses.cache.GetResponse
+import software.momento.kotlin.sdk.responses.cache.SetResponse
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class CacheClientScalarTest: BaseAndroidTestClass() {
```

Comment:
I copied the tests from the jvm cache client test here for now. I couldn't get putting the test methods in an abstract class to work, but I'm sure there is a way to avoid copying everything.